CHANGES.rst

```diff
@@ -195,6 +195,10 @@ Other Changes and Additions
     ``angles.angle_axis`` are also deprecated, in favour of the new routines
     with the same name in ``coordinates.matrix_utilities``. [#5104]
 
+- ``astropy.io.fits``
+
+  - Optimized the loading of tables with many columns. [#4985]
```

Comment:
I think "optimized" is a strong word.  It's hardly optimal.  You can just say "Performance improvements"